WordPress/Classes/ViewRelated/Tools/SettingsTextViewController.m

```diff
@@ -135,6 +223,10 @@ - (BOOL)textField:(UITextField *)textField shouldChangeCharactersInRange:(NSRang
         [self.navigationController popViewControllerAnimated:YES];
     }
 
+    if ([string isEqualToString:@"\n"]) {
```

Comment:
Worst part is that... three lines above, we've got the exact same check, using `NSCharacterSet` and `NSRange`. I honestly don't know what was i thinking about, thanks for catching this!!